test/setup/build.ts

```diff
@@ -0,0 +1,35 @@
+import { execSync } from 'child_process';
+import { renameSync, rmSync } from 'fs';
+import { projectRoot } from '../utils';
+
+const rustGithubRepository = 'metaplex-program-library';
+const repostitoryDir = `${rustGithubRepository}-master`;
+
+const rustProgramsRepository = `https://github.com/metaplex-foundation/${rustGithubRepository}/archive/refs/heads/master.zip`;
+
+function build() {
+  const programs: string[] = [
+    'auction/program',
+    'token-metadata/program',
+    'token-vault/program',
+    'metaplex/program',
+  ];
+
+  process.chdir('test');
+  rmSync(repostitoryDir, { recursive: true, force: true });
+  rmSync('rust', { recursive: true, force: true });
+  rmSync('./master.zip', { force: true });
+
+  execSync(`wget --quiet ${rustProgramsRepository}`);
```

Comment:
Please use a library from npm like `request` or `fetch` to perform this.
Never depend on any external tooling like `wget` to be installed on the dev's machine or CI environment.